products/sle/main.pm

```diff
@@ -1469,10 +1469,8 @@ elsif (get_var('HPC')) {
     }
     else {
         loadtest 'boot/boot_to_desktop';
-        if (check_var('HPC', 'enable')) {
-            loadtest 'hpc/enable_in_zypper';
-        }
-        elsif (check_var('HPC', 'conman')) {
+        loadtest 'hpc/enable_in_zypper' if(sle_version_at_least('15'));
```

Comment:
Why do we want to disable this one for non SLE12 and ignore HPC setting?

Reply:
you meant for non SLE15, right ? initially it was independent test which was running on SLE12 SP2. Now it is not run anywhere.  But still this if(get_var('HPC')) branch is used by QAM. Currently I am using this test to add additional repository and I want to do this only for SLE15 to not ruin QAM flow

Reply:
sle_version_at_least is tricky function and it returns true for all non sle distris. Another point is that we change behavior for something which was there and works (for SLE12, most likely). So I would rather apply SLE15 specific logic in the test itself. As @Soulofdestiny  has already merged this, please, do not forget to revert this change.

Reply:
but HPC tree will never be executed on non-sle

Reply:
I would say even more - we are inside products/sle/main.pm which is by design not capabale to run not on sle - using is_sle inside this file make no sense. 
Regarding SLE12 noone runs this test in SLE12 and I don't know about any plans in this regard so no point to keep this flow